### PR TITLE
fix: remove dependencies bug

### LINKED ISSUE #23: removePackageDependencies doesn't remove if there are multiple dependency types
Passing the dependency array within an object seems to not remove the dependency

// does not work
`
await removePackageDependencies({
    dependencies: [
        'myDependency'
    ]
});
`

// works
`
await removePackageDependencies([
    'myDependency'
]);
`

### PR DESCRIPTION
**Pull request by stekovinbranturry**
Fixes #23

### CODE REVIEW

**Comment by tommy-mitchell** [comment]
Can you add tests for the `await removePackageDependencies({ dependencies: [ 'myDependency' ] });` style usage?

**Comment by stekovinbranturry** [comment]
> Can you add tests for the `await removePackageDependencies({ dependencies: [ 'myDependency' ] });` style usage?

added

**Comment by sindresorhus** [comment]
It would be better to add a new test than modify existing tests.

**Comment by stekovinbranturry** [comment]
> It would be better to add a new test than modify existing tests.

`test('async - multiple types')` is literally for the case `await removePackageDependencies(Record<DependencyKey, Dependency[]>)`, do we really need another case?

File: test/remove-dependencies.js [hunk 1]
```diff
@@ -43,10 +43,10 @@ test('async - multiple types', async t => {
 
 	t.deepEqual(packageJson, omit(fixture, ['dependencies']));
 
-	await removePackageDependencies(temporary, {devDependencies: ['bar']});
+	await removePackageDependencies(temporary, {devDependencies: ['bar', 'baz'], peerDependencies: ['foobar']});
 	packageJson = await readPackage({cwd: temporary, normalize: false});
 
-	t.deepEqual(packageJson, omit(fixture, ['dependencies', 'devDependencies.bar']));
+	t.deepEqual(packageJson, pick(fixture, ['name']));
```
Comment (sindresorhus) [inline test/remove-dependencies.js:49]: Feels simpler to just do this:

```suggestion
	t.deepEqual(packageJson, {name: fixture.name}));
```

**Comment by sindresorhus** [comment]
The test already does too much. Large tests often hide bugs since they do too much. Not a bit issue in this case, but having smaller tests would make it clearer. I guess that could be done later though.

**Comment by stekovinbranturry** [comment]
> The test already does too much. Large tests often hide bugs since they do too much. Not a big issue in this case, but having smaller tests would make it clearer. I guess that could be done later though.

yes, it will be better, i've already add a new case

**Pull request merged**